### PR TITLE
UI: Rename Wild Arms hack

### PR DESCRIPTION
**Pull request by JordanTheToaster**
### Description of Changes
Renames the Wild Arms hack to something closer to what it actually does internally.

### Rationale behind Changes
More logical naming and less game related hacks good.

### Suggested Testing Steps
Make sure CI is happy the hack still functions and it loads properly from the DB in games that have it as a gamefix.

### CODE REVIEW

**Comment by lextra2** [comment]
The tooltip should be more descriptive of what it does.

And if done so, it should also mention something like
`Formerly called Wild Arms hack`
because people will wonder where it went.

File: pcsx2-qt/Settings/GraphicsSettingsWidget.ui [hunk 1]
```diff
@@ -1172,9 +1172,9 @@
           </widget>
          </item>
          <item row="1" column="1">
-          <widget class="QCheckBox" name="wildHack">
+          <widget class="QCheckBox" name="forceEvenSpritePosition">
            <property name="text">
-            <string extracomment="Wild Arms: name of a game series. Leave as-is or use an official translation.">Wild Arms Hack</string>
+            <string extracomment="Wild Arms: name of a game series. Leave as-is or use an official translation.">Force Even Sprite Position</string>
```
Comment (TellowKrinkle) [inline pcsx2-qt/Settings/GraphicsSettingsWidget.ui:1177]: You can probably remove the translator comment, since it no longer mentions wild arms

Comment (JordanTheToaster) [reply pcsx2-qt/Settings/GraphicsSettingsWidget.ui]: Can do.

File: pcsx2/Docs/GameIndex.md [hunk 2]
```diff
@@ -174,7 +174,7 @@ The clamp modes are also numerically based.
 
 * alignSprite                [`0` or `1`]                               {Off or On}                        Default: Off (`0`)
 * mergeSprite                [`0` or `1`]                               {Off or On}                        Default: Off (`0`)
-* wildArmsHack               [`0` or `1`]                               {Off or On}                        Default: Off (`0`)
+* forceEvenSpritePosition               [`0` or `1`]                               {Off or On}                        Default: Off (`0`)
```
Comment (lightningterror) [inline pcsx2/Docs/GameIndex.md:177]: Formatting, spacing should match the others, top/bottom ones.

**Review by lightningterror — COMMENTED** [review]
Perhaps it would've been better to split the commit in several: gs changes, qt, fsui, config, doc removal, db changes.

**Review by Mrlinkwii — APPROVED** [review]

**Review by kamfretoz — APPROVED** [review]

**Comment by kamfretoz** [comment]
Missed this during the testing, the text for the checkbox doesn't appear and the GameDB entry for it aren't getting loaded.
image
image

**Review by Mrlinkwii — APPROVED** [review]

**Pull request merged**